cvmfs/malloc_heap.cc

```diff
@@ -107,12 +107,7 @@ MallocHeap::MallocHeap(uint64_t capacity, CallbackPtr callback_ptr)
   assert((capacity_ % 8) == 0);
   heap_ = reinterpret_cast<unsigned char *>(sxmmap(capacity + 7));
   uintptr_t head = uintptr_t(heap_) % 8;
-  if (head > 0)
```

Comment:
I see, I think we can than simply run `sxmmap` and add an assert that the resulting pointer is 8 byte aligned.